features/generator_specs/generator_specs.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Generator spec
+
+    RSpec spec(s) can be generated when generating application components.
+    For instance, `rails generate model` will also generate an RSpec spec file
+    for the model but you can also write your own generator. See [customizing your workflow](https://guides.rubyonrails.org/generators.html#customizing-your-workflow)
```

Comment:
Sorry I meant all 3 lines should be wrapped together :joy:

Reply:
Oups. Changed in https://github.com/rspec/rspec-rails/pull/2217/commits/9dc292467de27552b4086096b00c003309fde0c6